### PR TITLE
update code to disable worload mcp

### PR DESCRIPTION
**Pull request by liqcui**
update code to disable workload mcp and update the wrong description in doc

### CODE REVIEW

**Comment by liqcui** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.16-nightly-x86-control-plane-3nodes

File: ci-operator/step-registry/openshift-qe/workers-infra-workload/openshift-qe-workers-infra-workload-commands.sh [hunk 1]
```diff
@@ -708,8 +708,10 @@ esac
 
 #Create infra and workload machineconfigpool
 create_machineconfigpool infra
-create_machineconfigpool workload
 
+if [[ $IF_CREATE_WORKLOAD_NODE == "true" ]];then
+create_machineconfigpool workload
```
Comment (qiliRedHat) [inline ci-operator/step-registry/openshift-qe/workers-infra-workload/openshift-qe-workers-infra-workload-commands.sh:713]:
```suggestion
  create_machineconfigpool workload
```

File: ci-operator/step-registry/openshift-qe/workers-infra-workload/openshift-qe-workers-infra-workload-ref.yaml [hunk 2]
```diff
@@ -45,7 +45,7 @@ ref:
   - name: IF_CREATE_WORKLOAD_NODE
     default: "false"
     documentation: |-
-      Default is true, which means openshift-qe-workers-infra-workload-commands.sh will create workload node. You can set it to false to disable creating workload node
+      Default is false, set it to true which means openshift-qe-workers-infra-workload-commands.sh will create workload node. set it to false to disable creating workload node
```
Comment (qiliRedHat) [inline ci-operator/step-registry/openshift-qe/workers-infra-workload/openshift-qe-workers-infra-workload-ref.yaml:48]:
```suggestion
      Default is false, set it to true to let openshift-qe-workers-infra-workload-commands.sh to create workload node. set it to false to disable creating workload node
```

**Pull request merged**